udata_hydra/routes/__init__.py

```diff
@@ -72,6 +73,7 @@ def generate_routes(
         delete_resource_exception,
         None,
     ),
+    (web.post, "/api/file-analysis/", analyse_file, None),
```

Comment:
By default, `post` requests require authentication.

It means that it won't be the user client itself that will call this route, but maybe we want to use udata or cdata api as proxy?